src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+"""Use DNS twist to fuzz domain names and cross check with a blacklist."""
+# Standard Python Libraries
+import datetime
+import json
+import pathlib
+import traceback
+
+# Third-Party Libraries
+import dnstwist
+import dshield
+import psycopg2.extras as extras
+import requests
+
+# cisagov Libraries
+from pe_reports import app
+
+from .data.pe_db.db_query_source import (
+    addSubdomain,
+    connect,
+    getDataSource,
+    getSubdomain,
+    org_root_domains,
+    query_orgs_rev,
+)
+
+date = datetime.datetime.now().strftime("%Y-%m-%d")
+LOGGER = app.config["LOGGER"]
+
+
+def checkBlocklist(dom, sub_domain_uid, source_uid, pe_org_uid, perm_list):
+    """Cross reference the dnstwist results with DShield Blocklist."""
+    malicious = False
+    attacks = 0
+    reports = 0
+    if "original" in dom["fuzzer"]:
+        return None, perm_list
+    elif "dns_a" not in dom:
+        return None, perm_list
```

Comment:
These conditions can be combined since they return the same thing:
```suggestion
    if "original" in dom["fuzzer"] or "dns_a" not in dom:
        return None, perm_list
```